torch/jit/_monkeytype_config.py

```diff
@@ -85,6 +85,10 @@ def consolidate_types(self, qualified_name: str) -> Dict:
                 if len(types) > 1:
                     all_args[arg] = {'Any'}
                 else:
+                    if 'NoneType' in _all_type:
+                        # Handle all Nonetype as Optional[torch.Tensor]
+                        # TODO: Handle Optional for other data types
+                        _all_type = _all_type.replace('NoneType', 'Optional[torch.Tensor]')
```

Comment:
This is not safe.

Any container that contains a `NoneType` would be changed into container of `Optional[torch.Tensor]`, which I think isn't what you intended.

Should we use MonkeyType TypeRewriter for this purpose?
https://monkeytype.readthedocs.io/en/stable/configuration.html#monkeytype.config.Config.type_rewriter

Reply:
Also, I am curious why it is necessary to rewrite `None`

Reply:
As discussed offline, now changed.